### PR TITLE
Update to macse v2.07

### PR DESCRIPTION
**Pull request by ranwez**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by ranwez** [comment]
@BiocondaBot please add label

**Review by bernt-matthias — COMMENTED** [review]
On line 16 change `number: 1` to `number: 0`
```

**Comment by ranwez** [comment]
@bioconda-bot please update

**Comment by ranwez** [comment]
@bernt-matthias Thanks for your help. I change the build number and ask for update

**Comment by ranwez** [comment]
@bioconda-bot please update

**Comment by ranwez** [comment]
The help message still contains MACSE. The option "-h" is not valid but any invalid option lead to printing the help message so it should be fine. I look at the lint detail and there is a strange line:
"11:16:14 BIOCONDA INFO (OUT) export PREFIX=/opt/conda/conda-bld/macse_1678792411303/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl..."

Regarding the python wrapper I have no idea how this work. I was naively hoping that I could get it work by using a slight adaptation of the previous recipe ...

I try commenting the test part as you suggested ...

**Comment by ranwez** [comment]
@bioconda-bot please update

**Comment by ranwez** [comment]
@bioconda-bot please update

**Comment by ranwez** [comment]
@bioconda-bot please update

**Comment by ranwez** [comment]
@bioconda-bot please update

**Comment by ranwez** [comment]
@bioconda-bot please update

File: recipes/macse/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
+{% set sha256hash = "96873a1465f1e1aa9d0c462d469eee6954d36e8639727f69f34a72fcb6583963" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   # url: https://bioweb.supagro.inra.fr/{{ name }}/releases/{{ name }}_v{{ version }}.jar
-  url: https://bioweb.supagro.inra.fr/macse/releases/macse_v1.2.jar
+  # url: https://github.com/ranwez/MACSE_V2_PIPELINES/blob/master/UTILS/macse_v2.07.jar?raw=true
```
Comment (bernt-matthias) [inline recipes/macse/meta.yaml:11]:
```suggestion
```

File: recipes/macse/meta.yaml [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
 requirements:
   run:
     - openjdk >=1.5
-    - python
 
 test:
   commands:
-    - macse -h 2>&1 | grep MACSE
-    - macse --version 2>&1 | grep MACSE
+#    - echo "dummy test MACSE" | grep MACSE
```
Comment (bernt-matthias) [inline recipes/macse/meta.yaml:25]:
```suggestion
```

File: recipes/macse/meta.yaml [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
     - openjdk >=1.5
-    - python
 
 test:
   commands:
-    - macse -h 2>&1 | grep MACSE
-    - macse --version 2>&1 | grep MACSE
+#    - echo "dummy test MACSE" | grep MACSE 
+    - macse -help 2>&1 | grep MACSE
+#    - macse --version 2>&1 | grep MACSE
```
Comment (bernt-matthias) [inline recipes/macse/meta.yaml:27]:
```suggestion
```

File: recipes/macse/build.sh [hunk 4]
```diff
@@ -1,11 +1,17 @@
...
-ln -s $outdir/macse $PREFIX/bin
+TGT="$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
+[ -d "$TGT" ] || mkdir -p "$TGT"
+[ -d "${PREFIX}/bin" ] || mkdir -p "${PREFIX}/bin"
+
+cd "${SRC_DIR}"
+# Do not install Linux specific x86-acceleration libraries
+if [ "$(uname)" == "Darwin" ]; then
+    rm -f libIntel*.so
+fi
```
Comment (bernt-matthias) [inline recipes/macse/build.sh:12]: I guess this also does not apply to macse? 

```suggestion
```

File: recipes/macse/meta.yaml [hunk 5]
```diff
@@ -1,29 +1,30 @@
...
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   # url: https://bioweb.supagro.inra.fr/{{ name }}/releases/{{ name }}_v{{ version }}.jar
-  url: https://bioweb.supagro.inra.fr/macse/releases/macse_v1.2.jar
+  # url: https://github.com/ranwez/MACSE_V2_PIPELINES/blob/master/UTILS/macse_v2.07.jar?raw=true
+  url: https://bioweb.supagro.inra.fr/macse/releases/macse_v2.07.jar
```
Comment (bernt-matthias) [inline recipes/macse/meta.yaml:12]:
```suggestion
    url: https://bioweb.supagro.inra.fr/macse/releases/macse_v2.07.jar
```

Comment (ranwez) [reply recipes/macse/meta.yaml:12]: Apparently the problem was related to the url jar path. I would have prefer to get the jar from github 
but replacing  url: https://github.com/ranwez/MACSE_V2_PIPELINES/blob/master/UTILS/macse_v2.07.jar?raw=true
by
 url: https://bioweb.supagro.inra.fr/macse/releases/macse_v2.07.jar
fix problems ....

Comment (ranwez) [reply recipes/macse/meta.yaml]: I have commited the suggested modification (cleaning yaml file). So everything should be fine now.

**Comment by ranwez** [comment]
@bioconda-bot please update

File: recipes/macse/meta.yaml (hunk 5 above)
Comment (ranwez) [reply recipes/macse/meta.yaml]: thanks again for your help

**Comment by bernt-matthias** [comment]
@BiocondaBot please add label

**Comment by ranwez** [comment]
@bernt-matthias Do I need to do something else for macse v2 to be released via biocondan::anaconda.org? sorry to bother you once again ...

**Comment by bernt-matthias** [comment]
Hi @ranwez. Someone has to merge. Then they become available automatically. I pinged the bioconda gitter channel.

In the worst case I can also merge. But I prefer to have another pair of eyes on this.

File: recipes/macse/build.sh [hunk 6]
```diff
@@ -1,11 +1,13 @@
...
 
-outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
-mkdir -p $outdir
-mkdir -p $PREFIX/bin
-cp -R * $outdir/
-cp $RECIPE_DIR/macse.py $outdir/macse
-ls -l $outdir
-ln -s $outdir/macse $PREFIX/bin
+TGT="$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
+[ -d "$TGT" ] || mkdir -p "$TGT"
```
Comment (marcelm) [inline recipes/macse/build.sh:5]: Why not leave this as `mkdir -p "${TGT}"`? With `-p`, there won’t be an error if the directory already exists. (I realize this is how Picard’s `build.sh` does it, but it’s not needed.)

**Review by marcelm — APPROVED** [review]
Some minor comments, but feel free to ignore and merge anyway.

**Comment by ranwez** [comment]
@bernt-matthias @marcelm 
Thanks for your feedback, I have updated the recipe accordingly. In the [recipe of macse 1.2](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/macse/macse.py) there is a particular case for *pass_args[0].startswith('eu')* and some instruction related to exec_dir. I don't really understand what these instructions were supposed to do and I'm wondering if this should be included in this updated recipe.

**Comment by bernt-matthias** [comment]
@BiocondaBot please merge

**Comment by ranwez** [comment]
I just tested the macse_v2.07 conda package via Anaconda and it works just fine! thanks!

**Pull request merged**